pkg/hookbot/hookbot.go

```diff
@@ -310,16 +324,20 @@ func (h *Hookbot) Del(l Listener) {
 
 // The topic is everything after the "/pub/" or "/sub/"
 // Do not capture the "/unsafe". See note in `Topic()`.
-var TopicRE *regexp.Regexp = regexp.MustCompile("^(?:/unsafe)?/[^/]+/(.*)$")
+var TopicRE *regexp.Regexp = regexp.MustCompile("^(?:/unsafe)?/(?:pub|sub)/(.*)$")
```

Comment:
You could make the two cases (`/` and `/(pub|sub)/`) explicit and let anything else error?